cupy/core/core.pyx

```diff
@@ -2303,3 +2337,172 @@ cdef _mean = create_reduction_func(
      ('e->e', (None, None, None, 'float')),
      'f->f', 'd->d'),
     ('in0', 'a + b', 'out0 = a / (_in_ind.size() / _out_ind.size())', None))
+
+# -----------------------------------------------------------------------------
+# scan
+# -----------------------------------------------------------------------------
+
+def _inclusive_scan_kernel(dtype, block_size):
```

Comment:
For example.
https://github.com/pfnet/chainer/blob/457550fc2501f3b8f73f93d69400f51d4096696f/cupy/core/reduction.pxi#L168